tests/integration-test/bpf/build_push_images.sh

```diff
@@ -35,3 +35,15 @@ docker build \
 
 docker push quay.io/bpfd-bytecode/tracepoint
 
+docker build \
+ --build-arg PROGRAM_NAME=mallocprobe \
+ --build-arg SECTION_NAME=malloc \
+ --build-arg PROGRAM_TYPE=uprobe \
+ --build-arg BYTECODE_FILENAME=mallocprobe.bpf.o \
+ --build-arg KERNEL_COMPILE_VER=$(uname -r) \
```

Comment:
Please remove the Kernel_Compile_Ver tag from here RE https://github.com/bpfd-dev/bpfd/pull/505/commits/125b7c357a496f3480ef4dc0c013afacb979ca77

Reply:
+ anywhere else you might have added it :)

Reply:
Ok.

Reply:
I also rebuilt and pushed the images.